src/encoded/types/analysis.py

```diff
@@ -0,0 +1,141 @@
+from snovault import (
+    calculated_property,
+    collection,
+    load_schema,
+)
+from .base import Item
+
+
+@collection(
+    name='analyses',
+    unique_key='accession',
+    properties={
+        'title': 'Analyses',
+        'description': 'Listing of analysis',
+    })
+class Analysis(Item):
+    item_type = 'analysis'
+    schema = load_schema('encoded:schemas/analysis.json')
+    name_key = 'accession'
+    embedded = [
+        'files',
+        'files.quality_metrics',
+        'pipeline_labs',
+    ]
+
+    @calculated_property(schema={
+        "title": "Assemblies",
+        "description": "A list of genome assemblies on which this analysis is performed.",
+        "comment": "Do not submit. This field is calculated from files in this analysis.",
+        "type": "array",
+        "notSubmittable": True,
+        "items": {
+            "type": "string"
+        }
+    })
+    def assemblies(self, request, files):
```

Comment:
isn't this guaranteed to be singular? Unless something has gone horribly awry?  I feel like we should do something like file.assay_term_name which uses take_one_or_return_none (could return some error code or "mixed" or something that audits could trap.

Same for genome_annotations